omas/machine_mappings/d3d.py

```diff
@@ -1753,6 +1753,109 @@ def wall(ods, pulse, EFIT_tree="EFIT01", EFIT_run_id=None):
     ods["wall.time"] = [0.0]
     ods["wall.ids_properties.homogeneous_time"] = 1
 
+# ================================
+@machine_mapping_function(__regression_arguments__, pulse=174436)
+def reflectometer_hardware(ods, pulse):
+    """
+    Loads DIII-D reflectometer chord locations
+
+    The chord endpoints are approximative. They do not take into account
+    the vessel wall contour of the shot. The values have been taken from IDA-lite.
+
+    Data sources: https://github.com/GA-IDA/ida_lite/blob/c1398c826b7a327d6629b5518c3219b8870436ce/D3D/synt_diags/REFL.py#L41
+
+    :param ods: an OMAS ODS instance
+
+    :param pulse: int
+    """
+    bands=['Q','V','QO', 'VO']
+
+    # reflectometer LOS, not actual location of measured density!
+    R_in, R_out = 1.6, 2.377
```

Comment:
2.377 is from reflectometer documentation, is is the location where the phase is set to be zero.